nemo_text_processing/inverse_text_normalization/en/taggers/cardinal.py

```diff
@@ -31,10 +34,14 @@ class CardinalFst(GraphFst):
     Finite state transducer for classifying cardinals
         e.g. minus twenty three -> cardinal { integer: "23" negative: "-" } }
     Numbers below thirteen are not converted.
+
+    Args:
+        input_case: accepting either "lower_cased" or "cased" input.
     """
 
-    def __init__(self):
+    def __init__(self, input_case: str):
```

Comment:
to make backward compatible can we for now use default of lower_cased?

Reply:
done